src/simulator/layout/layouts/grid.ts

```diff
@@ -0,0 +1,28 @@
+import { IEdgeBase } from '../../../models/edge';
+import { INode, INodeBase, INodePosition } from '../../../models/node';
+import { ILayout } from '../layout';
+
+export class GridLayout<N extends INodeBase, E extends IEdgeBase> implements ILayout<N, E> {
+  constructor(private width: number, private height: number) {
+    this.width = width;
+    this.height = height;
+  }
+
+  getPositions(nodes: INode<N, E>[]): INodePosition[] {
+    const rows = Math.ceil(Math.sqrt(nodes.length));
+    const cols = Math.ceil(nodes.length / rows);
+    const cellWidth = this.width / cols;
+    const cellHeight = this.height / rows;
+
+    const positions: INodePosition[] = [];
+    nodes.forEach((node, index) => {
```

Comment:
Would a grid layout create such a layout where there is a minimum number of edge crosses? Would that be the best grid layout?